adder-codec-rs/Cargo.toml

```diff
@@ -22,7 +22,8 @@ default-run = "adder_simulproc"
 default = ["transcoder"]
 transcoder = ["opencv", "dep:fast-math"]
 raw-codec = []
-docs-only = ["opencv/docs-only", "dep:fast-math"]
+#docs-only = ["opencv/docs-only", "dep:fast-math"]
+docs-only = ["dep:fast-math"]
```

Comment:
Must still have opencv/docs-only for Cargo docs to generate correctly.

Reply:
About that, [that feature](https://github.com/twistedfall/opencv-rust/commit/a8108c52e0ae668b3abb256dd0bcccf41c7b1688) was removed from opencv-rust.